x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/persistence/DataFrameAnalyticsConfigProvider.java

```diff
@@ -63,7 +63,6 @@
 
     static {
         Map<String, String> modifiable = new HashMap<>();
-        modifiable.put(ToXContentParams.INCLUDE_TYPE, "true");
         modifiable.put(ToXContentParams.FOR_INTERNAL_STORAGE, "true");
         TO_XCONTENT_PARAMS = Collections.unmodifiableMap(modifiable);
     }
```

Comment:
Now this only has one entry this whole `static` block can be removed and the declaration above changed to simply:

```
private static final Map<String, String> TO_XCONTENT_PARAMS = Collections.singletonMap(ToXContentParams.FOR_INTERNAL_STORAGE, "true");
```

Reply:
Update code to reflect this.